### DISCUSSION TITLE
Cfclient doesn't recognize/detect Lighthouse V2 base

### DISCUSSION THREAD

**Discussion by daffakanz** [bitcraze/discussions#1543] [Q&A] [answered]
Hello, I'm a beginner to drones and robotic in general.
I'm trying out crazyflies to do simple autonomous flying. 

I followed the official step in the web. 
https://www.bitcraze.io/documentation/tutorials/getting-started-with-lighthouse/
also used the Youtube video as reference.
https://www.youtube.com/watch?v=DCEHht72B08

Problem starts here.
Right now, I'm stuck at "Select system type" step. My client doesn't connect with any of the Base Stations.
![WhatsApp Image 2024-08-27 at 14 17 33_2200c813](https://github.com/user-attachments/assets/371343ed-a551-4690-a321-c9d74181b219)

When I set BS channel, It connects and I successfully changed the channel to 1 and 2. Just like in the tutorial.
I re-flashed my Crazyflie just in case the problem was from the expansion deck. But the client still doesn't want to connect with the BS.
Did I do something wrong in between the steps?

This is my initial setup.
![WhatsApp Image 2024-08-27 at 14 17 52_c217904a](https://github.com/user-attachments/assets/805a3447-f715-4d36-8845-38263a831069)
And this is BS status.
![WhatsApp Image 2024-08-27 at 14 18 37_5b27cdba](https://github.com/user-attachments/assets/5eead8fe-95a2-417e-a942-4842aaf760ca)

**Comment by ataffanel** [comment, marked as answer]
Hi,

The Lighthouse deck looks installed in the wrong direction on your photo: there is an arrow on the top of the deck that shows the front of the deck, this seems to be pointing toward the back of the drone on your photo.

You should first check that the deck is working fine, if it does, the red LED will not be lit  (it will light up at startup and then it should be off) and the yellow or green led should be lighting up. Then, you should be able to see more information in the lighthouse tab of the client.

**Reply by daffakanz** [reply]
Thank you for the answer.

I see, I had to rush when I installed the deck.

Can I continue this thread if there's still something wrong?

**Reply by ataffanel** [reply]
If the title still matches your problem feel free to continue the thread, otherwise please make a new thread, they are free and keeping the title accurate helps a lot when others will have the same or similar issues later :-).
